packages/connector-types/jest.config.ts

```diff
@@ -0,0 +1,20 @@
+import type { Config } from '@jest/types';
```

Comment:
@darcyYe @simeng-li @wangsijie we need a shared jest config i think. mind to add an issue?

Reply:
[LOG-2271](https://linear.app/silverhand/issue/LOG-2271/add-a-shared-jest-config) was added to track this.

Reply:
@darcyYe cool, please see #644

Reply:
good suggestion, BTW @darcyYe  do we need to process `'^jose/(.*)$': '<rootDir>/node_modules/jose/dist/node/cjs/$1',` for connector package?  Does it include `jose` as its dependency?